fast/stages/2-networking-c-nva/test-resources.tf

```diff
@@ -14,242 +14,112 @@
-#       apt install iputils-ping bind9-dnsutils
-#     EOF
-#   }
-# }
-
-# module "test-vm-dev-secondary-0" {
-#   source     = "../../../modules/compute-vm"
-#   project_id = module.dev-spoke-project.project_id
-#   zone       = "${var.regions.secondary}-a"
-#   name       = "test-vm-dev-sec-0"
-#   network_interfaces = [{
-#     network = module.dev-spoke-vpc.self_link
-#     # change the subnet name to match the values you are actually using
-#     subnetwork = module.dev-spoke-vpc.subnet_self_links["${var.regions.secondary}/dev-default"]
-#   }]
-#   tags                   = ["secondary", "ssh"]
-#   service_account_create = true
-#   boot_disk = {
-#     initialize_params = {
-#         image = "projects/debian-cloud/global/images/family/debian-10"
-#     }
-#   }
-#   options = {
-#     spot               = true
-#     termination_action = "STOP"
-#   }
-#   metadata = {
-#     startup-script = <<EOF
-#       apt update
-#       apt install iputils-ping bind9-dnsutils
-#     EOF
-#   }
-# }
-
-# # Prod spoke
-
-# module "test-vm-prod-primary-0" {
-#   source     = "../../../modules/compute-vm"
-#   project_id = module.prod-spoke-project.project_id
-#   zone       = "${var.regions.primary}-b"
-#   name       = "test-vm-prod-pri-0"
-#   network_interfaces = [{
-#     network = module.prod-spoke-vpc.self_link
-#     # change the subnet name to match the values you are actually using
-#     subnetwork = module.prod-spoke-vpc.subnet_self_links["${var.regions.primary}/prod-default"]
-#   }]
-#   tags                   = ["primary", "ssh"]
-#   service_account_create = true
-#   boot_disk = {
-#     initialize_params = {
-#         image = "projects/debian-cloud/global/images/family/debian-10"
-#         type  = "pd-balanced"
-#         size  = 10
-#     }
-#   }
-#   options = {
-#     spot               = true
-#     termination_action = "STOP"
-#   }
-#   metadata = {
-#     startup-script = <<EOF
-#       apt update
-#       apt install iputils-ping bind9-dnsutils
-#     EOF
-#   }
-# }
-
-# module "test-vm-prod-secondary-0" {
-#   source     = "../../../modules/compute-vm"
-#   project_id = module.prod-spoke-project.project_id
-#   zone       = "${var.regions.secondary}-a"
-#   name       = "test-vm-prod-sec-0"
-#   network_interfaces = [{
-#     network = module.prod-spoke-vpc.self_link
-#     # change the subnet name to match the values you are actually using
-#     subnetwork = module.prod-spoke-vpc.subnet_self_links["${var.regions.secondary}/prod-default"]
-#   }]
-#   tags                   = ["secondary", "ssh"]
-#   service_account_create = true
-#   boot_disk = {
-#     initialize_params = {
-#         image = "projects/debian-cloud/global/images/family/debian-10"
-#     }
-#   }
-#   options = {
-#     spot               = true
-#     termination_action = "STOP"
-#   }
-#   metadata = {
-#     startup-script = <<EOF
-#       apt update
-#       apt install iputils-ping bind9-dnsutils
-#     EOF
-#   }
-# }
+output "ping_commands" {
```

Comment:
Move to outputs.tf?

Reply:
I thought about it and I don't mind having this specific output in this file, which makes it easier to delete it altogether - but the same should also apply to its variable which would make me a bit more nervous.

wdyt?

Reply:
I would either place both output and variable in the file, or put them in variables.tf and outputs.tf. Both work from my PoV, but I would be consistent.

Reply:
`tfdoc.py` has rightfully decided that things should exist where they are supposed to exist (i.e. misplaced outputs and variables are not tracked in `README.md`), so moving I'm moving the output to outputs.tf

Reply:
tfdocs reads any variables*.tf and outputs*tf but it seems a bit overkill to have separate files for test vms. Good point, agree on moving them like J suggested.